apps/meteor/client/components/message/toolbar/items/actions/Timestamp/TimestampPicker/TimestampPicker.spec.tsx

```diff
@@ -0,0 +1,123 @@
+import { mockAppRoot } from '@rocket.chat/mock-providers';
+import { composeStories } from '@storybook/react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import * as stories from './TimestampPicker.stories';
+import { TimestampPicker } from './index';
+import { dateToISOString, generateTimestampMarkup } from '../../../../../../../lib/utils/timestamp/conversion';
+import { TIMESTAMP_FORMATS } from '../../../../../../../lib/utils/timestamp/formats';
+
+jest.mock('../../../../../../GazzodownText', () => ({
+	__esModule: true,
+	default: () => <div data-testid='mock-gazzodown-text'>Mocked GazzodownText</div>,
+}));
+
+const wrapper = mockAppRoot().withUserPreference('useEmojis', true).withSetting('UI_Use_Real_Name', false).withJohnDoe();
+
+const testCases = Object.values(composeStories(stories)).map((Story) => [Story.storyName || 'Story', Story]);
+
```

Comment:
Let's create some accessibility tests, for that we use the [jest-axe](https://www.npmjs.com/package/jest-axe) package.

Here is an example https://github.com/RocketChat/Rocket.Chat/blob/develop/apps/meteor/client/views/audit/components/SecurityLogDisplayModal.spec.tsx#L18

Basically iterate over all stories and then pass the rendered component to the `axe()` function